include/eve/module/special/regular/double_factorial.hpp

```diff
@@ -48,11 +48,11 @@ namespace eve
 //!   namespace eve
 //!   {
 //!      // Regular overload
-//!      constexpr auto double_factorial(unsigned_value auto x)                          noexcept; // 1
+//!      template <unsigned_value T> constexpr as_wide_as_t<double,T>double_factorial(T x) noexcept; // 1
```

Comment:
Should be : 
`template<unsigned_value T> constexpr as_wide_as_t<double,T> double_factorial(T x)`

Missing space before `double_factorial`